From: SeongJae Park <sj@kernel.org>
Date: Sun, 4 Jan 2026 09:53:26 -0800
Subject: [PATCH] Docs/admin-guide/mm/damon/usage: fix wrong special purpose
 modules reference

It was unnecessarily having '_' at the end of the reference.  It was
causing 'make htmldocs' warning.  Fix it.

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202601040942.MWKEmMKA-lkp@intel.com/
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/admin-guide/mm/damon/usage.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/admin-guide/mm/damon/usage.rst b/Documentation/admin-guide/mm/damon/usage.rst
index f06bf260c22d..d3f4a0e89dc2 100644
--- a/Documentation/admin-guide/mm/damon/usage.rst
+++ b/Documentation/admin-guide/mm/damon/usage.rst
@@ -7,7 +7,7 @@ Detailed Usages
 DAMON provides below interfaces for different users.
 
 - *Special-purpose DAMON modules.*
-  :ref:`This <damon_modules_special_purpose>`_ is for people who building,
+  :ref:`This <damon_modules_special_purpose>` is for people who building,
   distributing, and/or administrating the kernel with special-purpose DAMON
   usages.  Using this, users can use DAMON's major features for the given
   purposes in build, boot, or runtime in simple ways.
-- 
2.47.3

